tools/log_collection/collect_archive_logs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# save directory from which command is initiated
+collect_logs_directory=$(pwd)
+
+# make the vmdb/log directory the current directory
+vmdb_logs_directory="/var/www/miq/vmdb"
+pushd ${vmdb_logs_directory}
+
+# eliminiate any prior collected logs to make sure that only one collection is current
+rm -f log/evm_full_archive_$(uname -n)* log/evm_archived_$(uname -n)*
```

Comment:
I have made some changes to this in the source repo that go along with this line, and what I see as possibly a bug.  See [that PR](https://github.com/redhat-cfme-support/cloudforms_4.X_log_collection/pull/15) for reference.

In short, in both scripts, it seems like we aren't cleaning up everything we should be.